lib/billing/request.ts

```diff
@@ -110,7 +113,11 @@ export function request(config: ConfigInterface) {
       });
     }
 
-    return data.confirmationUrl;
+    if (returnObject) {
+      return data as BillingRequestResponseObject;
```

Comment:
Could we just type `data` as this in line 74?

Reply:
`BillingRequestResponseObject` doesn't have a `userErrors` property, which `RequestResponse` does...